packages/backend/.default.env

```diff
@@ -9,6 +9,7 @@ FIRST_ZKEY_INDEX="00000"
 TIMEOUT_TOLERANCE_RATE=20 # average time plus 20% of that time.
 BC_RETRY_WAITING_TIME_IN_DAYS=3 # time to pass between a (blocking contribution) timeout and a retry.
 CF_RETRY_WAITING_TIME_IN_DAYS=1 # time to pass between a (blocking cloud function) timeout and a retry.
+CONFIG_CEREMONY_BUCKET_POSTFIX="-mpc-dev"
```

Comment:
Remove default value from the variable to avoid mismatch with `phase2cli` package configuration. (for example, I'm using `-ph2-ceremony` as postfix)